google/cloud/pubsub/internal/tracing_batch_sink_test.cc

```diff
@@ -216,6 +217,33 @@ TEST(TracingBatchSink, PublishSpanHasAttributes) {
                                  TestTopic().FullName())))));
 }
 
+#if OPENTELEMETRY_VERSION_MAJOR >= 1
+#if OPENTELEMETRY_VERSION_MINOR >= 13
+TEST(TracingBatchSink, PublishSpanIsRoot) {
+  auto span_catcher = InstallSpanCatcher();
+  auto message_span = MakeSpan("test span");
```

Comment:
Don't we want to mark this span as active to have a conclusive test?

Reply:
Yup. Done.